docs/gitbook/setup/README.md

```diff
@@ -5,8 +5,7 @@ To get up and running developing with Plasma, follow these instructions:
 #### Important note
 
 ````
-Select and Datepicker components do not work right now.
-Use react-select and datepicker directly in Spacestation.
+Select component does not work right now. Use react-select directly in Spacestation.
```

Comment:
Can we remove the select code as it is not used anywhere (and all commented out where it is used)